src/nginxconfig/i18n/zh-tw/templates/app.js

```diff
@@ -29,6 +29,9 @@ import common from '../common';
 export default {
     title: `${common.nginx} 配寘`,
     description: `配寘高性能、安全、穩定的${common.nginx}服務器的最簡單方法。`,
+    en: '英語', // TODO: translate
+    zhCN: '簡體中文', // TODO: translate
+    zhTW: '繁體中文', // TODO: translate
```

Comment:
👋 @moniang I was wondering if you could check these translations are correct (or suggest new ones)?

The source strings are:
```
    en: 'English',
    zhCN: 'Chinese (simplified)',
    zhTW: 'Chinese (traditional)',
```

Reply:
yes,no problem

Reply:
Yes, no problem